examples/lin_reg/src/main.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 MesaTEE Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use lazy_static::lazy_static;
+use mesatee_sdk::{Mesatee, MesateeEnclaveInfo};
+use serde_derive::Serialize;
+use serde_json;
+use std::net::SocketAddr;
+use std::{env, fs};
+
+lazy_static! {
+    static ref TMS_ADDR: SocketAddr = "127.0.0.1:5554".parse().unwrap();
+    static ref TDFS_ADDR: SocketAddr = "127.0.0.1:5065".parse().unwrap();
+}
+
+#[derive(Serialize)]
+pub(crate) struct LinRegPayload {
+    input_mode_columns: usize,
+    input_mode_data: String,
+    target_mode_data: String,
+    test_data: String,
+}
+
+fn print_usage() {
+    let msg = "
+    ./lin_reg input_mode_data_columns input_mode_data_path target_mode_data_path test_data_path 
+    input_model format:
+        f32,f32,f32,f32 ...
+        f32,f32,f32,f32 ...
+        ...
+
+    target_data format:
+        1.
+        0.
+        1.
+        0.
+        0.
+        1.
+        ....
+    
+    test_data format:
+        f32,f32,f32,f32 ...
+    ";
+    println!("usage: \n{}", msg);
```

Comment:
Clippy complaints about this line. See: https://ci.mesalock-linux.org/mesalock-linux/mesatee/40/5/5